tests/instruments/test_all_instruments.py

```diff
@@ -105,3 +228,15 @@ def test_kwargs_to_adapter(cls):
     with pytest.raises(ValueError,
                        match="'kwarg_test' is not a valid attribute for type SerialInstrument"):
         cls(SIM_RESOURCE, visa_library='@sim', kwarg_test=True)
+
+
+@pytest.mark.parametrize("prop_set", properties)
+def test_property_docstrings(prop_set):
+    device, property_name, prop = prop_set
+    if device.__name__ in legacy_instruments:
+        pytest.skip(f"{device.__name__} is in the codebase and has to be refactored later on.")
+    start = prop.__doc__.split(maxsplit=1)[0]
+    assert start in ("Control", "Measure", "Set", "Get"), (
```

Comment:
We should think about the choices we want to encourage. It would be easier to just have 3 choices to mirror the control/setting/measurement triad (also thinking of future parsing effort e.g. in pyleco). In that case control/set/read or control/set/get seem most straightforward. 
However, that might feel too restrictive. :shrug: ? The first thing I'd feel like extending is to allow more words for getting ("control" and "set" are already pretty clear and not "overloaded" as terms, I think). "Measure" and "Get" or "Measure" and "Read" (considering we are _reading_ from a device) seem most suitable.
Thoughts?

Reply:
Also, whatever we decide needs to be reflected in the [docs](https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings).

Reply:
> Also, whatever we decide needs to be reflected in the [docs](https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings).

I wrote the test to reflect the docs (except for the fact, that we do not verify the correct type, e.g. we do not catch "Set" for a `control`). I thought "Get" would be allowed for `measurement` as well, but now I see, it is not.

For a getting property, I think we should allow a more generic form than "Measure". It seems, for example, strange to use "Measure the serial number".

Reply:
"Measure" and "Read", then?

Reply:
OK, "Get" and "Measure", then? 
Get/set is nicely symmetric -- for "read" I'd expect "write" as the counterpart.

Reply:
> OK, "Get" and "Measure", then?

So: Get, Set, Measure, and Control.

Reply:
I adjusted the documentation accordingly.

Reply:
Yes, thanks!